File: energybalance_pkg/test/f90/test_potentialtranspiration.f90
!Test generation

PROGRAM test
    USE Potentialtranspirationmod
    REAL:: evapoTranspiration
    REAL:: tau
    REAL:: potentialTranspiration
    print *, "--------test_test1_PotentialTranspiration-------"
    tau = 0.9983
    evapoTranspiration = 830.958
    call model_potentialtranspiration(evapoTranspiration, tau,  &
            potentialTranspiration)
    print *, potentialTranspiration

END PROGRAM

File: energybalance_pkg/test/py/test_potentialtranspiration.py
'Test generation'

from Potentialtranspiration import *
from math import *
import numpy as np



def test_test1():
    params= model_potentialtranspiration(
    tau = 0.9983,
    evapoTranspiration = 830.958,
     )
    potentialTranspiration_estimated = round(params, 3)
    potentialTranspiration_computed = 1.413
    assert (potentialTranspiration_estimated == potentialTranspiration_computed)